src/Nowruz/pages/jobs/detail/applicants/useApplicantAction.tsx

```diff
@@ -300,5 +358,12 @@ export const useApplicantAction = (
     setOpenSelectedRejectAlert,
     openSelectedRejectAlert,
     handleRejectMultiple,
+    searchTerm,
+    page: pages.find(p => p.tab === currentTab)?.page || 1,
+    handleChangePage,
+    applicantsList,
+    total: totalCounts.find(item => item.tab === currentTab)?.count || 10,
```

Comment:
the default value of total is `10`?

Reply:
@satanmourner we always have the total value, it is just fir fixing typescript errors

Reply:
@marjan-ara yeah but I want to know, is it ok to be `10` when we have no data? (the total is `0` and the default value should be `0`)